tests/unit/test_util.py

```diff
@@ -37,32 +37,46 @@ class FakeModel(FakeApiModelExtension):
     name: str
 
 
+class FakeModel2(FakeApiModelExtension):
+    code: int = 0
+    name: str
+    status: str
+
+
 class TestParseInput:
 
     def test_no_data(self, app):
         """
         GIVEN a request with no parameters of any type
         WHEN we call parse_input on that request
-        THEN we get a BaseApiModel returned with a code of 400
+        THEN we get a BaseApiView returned with a code of 400
         AND an error explaining what is wrong
         """
         with app.test_request_context():
             retval = parse_input(app.logger, FakeModel)
-            assert 'Field required' in retval.error
-            assert isinstance(retval, BaseApiModel)
+            assert (len(retval.errors)) == 1
+            assert 'name' == [*retval.errors][0]
```

Comment:
suggestion: I would change this to:
```
assert 'name' in retval.errors
```
As currently written, this would throw a KeyError rather than an assertion error if the errors dict does not have the expected structure.

Reply:
Changed. Thanks!